src/main/java/io/cdap/plugin/salesforce/plugin/connector/SalesforceConnector.java

```diff
@@ -88,7 +88,7 @@ public void test(ConnectorContext connectorContext) throws ValidationException {
 
   @Override
   public BrowseDetail browse(ConnectorContext connectorContext, BrowseRequest browseRequest) throws IOException {
-    return browse();
+    return browse(false);
```

Comment:
Should queryableOnly be passed as true here to ensure only the queryable objects are returned?

Reply:
As per the current implementation, we do return all. Hence kept it false to avoid any regression.